lib/cretonne/src/ir/funcname.rs

```diff
@@ -52,7 +52,7 @@ impl FunctionName {
 fn try_as_name(bytes: &[u8]) -> Option<String> {
     let mut name = String::with_capacity(bytes.len());
     for c in bytes.iter().map(|&b| b as char) {
-        if c.is_ascii() && c.is_alphanumeric() || c == '_' {
+        if c.is_ascii() && c.is_alphanumeric() || c == '_' || c == '.' || c == '-' {
```

Comment:
If you think so I can remove it. This was really important for me to debug my translations, otherwise every function name was #065065AB5468DC5664 and I couldn't figure out where I was in the module.